kernel/kselftest.py

```diff
@@ -60,7 +60,7 @@ def setUp(self):
             deps = deps + rdeps
         elif 'SuSE' in detected_distro.name:
             deps = deps + sdeps
-        elif 'centos' in detected_distro.name:
+        elif detected_distro.name in ['centos', 'fedora']:
             deps = deps + cdeps
```

Comment:
it is already in older code  i think it denote for release like cdeps for centos dependency , any will rectify this